pkg/prove/querier.go

```diff
@@ -0,0 +1,56 @@
+package prove
+
+import (
+	"fmt"
+	"strconv"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/pkg/consts"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	"github.com/tendermint/tendermint/types"
+)
+
+const TxInclusionQueryPath = "txInclusionProof"
+
+// Querier defines the logic performed when the ABCI client using the Query
+// method with the custom prove.QueryPath. The index of the transaction being
+// proved must be appended to the path. The marshalled bytes of the transction
+// proof (tmproto.TxProof) are returned.
+//
+// example path for proving the third transaction in that block:
+// custom/tx-inclusion-proof/3
```

Comment:
[question] which fragment of the path is correct `tx-inclusion-proof` or `txInclusionProof`?

Reply:
good catch! 28da36536c35c8e488f108ee77b72dcb9b5d1b1c